opentelemetry-api/src/opentelemetry/trace/sampling.py

```diff
@@ -118,11 +175,15 @@ def should_sample(
         return Decision(trace_id & self.TRACE_ID_LIMIT < self.bound)
 
 
-# Samplers that ignore the parent sampling decision and never/always sample.
 ALWAYS_OFF = StaticSampler(Decision(False))
+"""Sampler that never samples spans, regardless of the parent span's sampling decision."""
```

Comment:
does this result in the variables being documented in sphinx? Just unfamiliar with that convention.

Reply:
yes, and I need them to be documented to reference them in the sampler doc